README.md

```diff
@@ -330,6 +330,20 @@ containing the files if more than one file is given (a new line character is
 introduced between each file). Wildcard `*` accepted.
 
 
+### head([{'-n', <num>},] file [, file ...])
+### head([{'-n', <num>},] file_array)
```

Comment:
Sorry if this is stupid, but what does head do? And maybe we should include a description here.